BuildProject/package.json

```diff
@@ -13,22 +13,22 @@
   },
   "dependencies": {
     "@promotedai/react-native-metrics": "file:../",
-    "appium": "^1.21.0",
+    "appium": "^1.8.1",
```

Comment:
This downgrade is on purpose: The latest version of Appium (1.21.0) has an open vulnerability.